content/client-server-api/modules/report_content.md

```diff
@@ -5,9 +5,6 @@ Users may encounter content which they find inappropriate and should be
 able to report it to the server administrators or room moderators for
 review. This module defines a way for users to report content.
 
-Content is reported based upon a negative score, where -100 is "most
-offensive" and 0 is "inoffensive".
-
```

Comment:
I removed this here because it only applies to event reporting and is repeated in the corresponding endpoint's documentation.